template/src/lib/sdk/requestsUtils.js

```diff
@@ -19,9 +19,7 @@ export function getRequest(url) {
  * Simple POST request
  */
 export function postRequest(url, payload) {
-  // TODO: Temporal Hack to bypass cors SQL API
-  const hackUrl = `https://cors-anywhere.herokuapp.com/`;
-  return new Request(`${hackUrl}${url}`, {
+  return new Request(url, {
```

Comment:
If I'm not wrong it will break the demo if you click on the widget category at the KPI section because the URL is larger than 2046 and CORS is not enable.

We should remove it, once backend fix it.

Reply:
Yeah, we have to wait. 👍